package.json

```diff
@@ -1,43 +1,45 @@
 {
-    "name": "trivia-bot",
-    "version": "1.0.1",
-    "description": "## Team Members",
-    "main": "bot.js",
-    "engines": {
-        "node": "16.9.1",
-        "npm": "8.12.2"
-    },
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "babel-node": "babel-node --presets='@babel/preset-env'",
-        "start": "babel-node src/bot.js",
-        "dev": "nodemon --exec npm run babel-node -- src/bot.js"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/elenirotsides/Trivia-Bot.git"
-    },
-    "author": "",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/elenirotsides/Trivia-Bot/issues"
-    },
-    "homepage": "https://github.com/elenirotsides/Trivia-Bot#readme",
-    "dependencies": {
-        "axios": "^0.27.0",
-        "discord.js": "^13.8.0",
-        "dotenv": "^16.0.0",
-        "glob": "^8.0.0",
-        "parse-entities": "^4.0.0",
-        "path": "^0.12.7",
-        "topgg-autoposter": "^2.0.1",
-        "util": "^0.12.4"
-    },
-    "devDependencies": {
-        "@babel/core": "7.18.13",
-        "@babel/node": "7.18.10",
-        "@babel/preset-env": "7.18.10",
-        "nodemon": "2.0.19"
-    },
-    "type": "module"
+  "name": "trivia-bot",
+  "version": "1.0.1",
+  "description": "## Team Members",
+  "main": "bot.js",
+  "engines": {
+    "node": "18.14.1",
```

Comment:
Is there a reason why you changed the required version? Does the app not work on v16.9.1 with the updates? @alanschapira @elijahd123